config/services.yaml

```diff
@@ -51,6 +51,10 @@ services:
     # add more service definitions when explicit configuration is needed
     # please note that last definitions always *replace* previous ones
 
+    App\EntityListener\:
+        resource: "../src/EntityListener"
```

Comment:
What about `App\EventListener\Doctrine` ?

Reply:
I prefer to keep `EntityListener` directory to be more consistent with Symfony architecture: https://symfony.com/doc/current/doctrine/events.html#doctrine-entity-listeners